src/Qir/Tests/QIR-dynamic/qsharp/qir-test-assert.qs

```diff
@@ -1,15 +1,363 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+// https://docs.microsoft.com/en-us/azure/quantum/concepts-the-qubit#single-qubit-operations    Qubit and Operations
+// https://docs.microsoft.com/en-us/azure/quantum/user-guide/libraries/standard/prelude         Operation Details, Multi-Qubit Operations
+
 namespace Microsoft.Quantum.Testing.QIR  {
 
     open Microsoft.Quantum.Diagnostics;
+    open Microsoft.Quantum.Intrinsic;
+
+    // To avoid the potentially circular dependency on QuantumLibraries repo we redefine the following {
```

Comment:
Fixing.
FYI: So far I haven't heard about the `internal` keyword in Q#. Probably the reason for that is because the "Callable Declarations" section does not mention `internal`. However the page has a link [Access modifiers](https://docs.microsoft.com/en-us/azure/quantum/user-guide/language/programstructure/accessmodifiers#access-modifiers) which takes to an empty page "Access modifiers in Q#".
I also fail to quickly find `internal` in the Q# docs.
https://github.com/microsoft/qsharp-language/issues/94